index.html

```diff
@@ -0,0 +1,102 @@
+        <section class="section-one">
+          <div class="billboard"></div>
+          <div class="content">
+            <nav class="top-nav">
+              <ul>
+                <li><a href="#" class="apple-icon"><i class="fab fa-apple apple"></i></a></li>
+                <li><a href="#"><span>Store</span></a></li>
+                <li><a href="#"><span>Mac</span></a></li>
+                <li><a href="#"><span>iPod</span></a></li>
+                <li><a href="#"><span>iPhone</span></a></li>
+                <li><a href="#"><span>iPad</span></a></li>
+                <li><a href="#"><span>iTunes</span></a></li>
+                <li><a href="#"><span>Support</span></a></li>
+                <li class="search-list">
+                  <div class="search">
+                    <form class="form-search" action="action_page.php">
+                      <div class="search-input">
+                        <input type="text" placeholder="" name="search">
+                        <button type="submit"><i class="fa fa-search search-icon"></i></button>
+                      </div>
+                    </form>
+                  </div>
+                </li>
+              </ul>
+            </nav>
+            <header class="header">
+              <div class="heading">
+                <h1><a href="#">What will your verse be?</a></h1>
+              </div>
+              <div class="sub-heading">
+                <h2><a href="#">Everyone has something to share. See how people are using iPad in their unique ways to add to the world's story.</a></h2>
+                <h2 class="lower"><a href="#">Watch the "Your Verse" film and explore the stories <span><i class="far fa-play-circle"></i></span></a></h2>
+              </div>
+            </header>
+          </div>
+        </section>
+
+        <section class="section-two">
+          <h5 hidden>Gallery</h5>
+          <div class="content">
+            <div class="images">
+              <div class="items">
+                <a href="#"><img src="https://web.archive.org/web/20140301004610im_/http://images.apple.com/home/images/promo_iphone5s.jpg" alt="iphone5"></a>
+              </div>
+              <div class="items">
+                <a href="#"><img src="https://web.archive.org/web/20140301004610im_/http://images.apple.com/home/images/promo_iphone5c.jpg" alt="iphone5"></a>
+              </div>
+              <div class="items">
+                <a href="#"><img src="https://web.archive.org/web/20140301004610im_/http://images.apple.com/home/images/promo_30_years.jpg" alt="Happy Birthday, Mac"></a>
+              </div>
+              <div class="items">
+                <a href="#"><img src="https://web.archive.org/web/20140301004610im_/http://images.apple.com/home/images/promo_supplier_responsibility.jpg" alt="Supplier Responsibilty"></a>
+              </div>
+            </div>
+          </div>
+        </section>
+
+        <footer>
+          <div class="links">
```

Comment:
* [x] It should be wrapped by `<footer>` tags
* [x] It includes links like copyright, terms of use, and privacy policy